data_hub_api/docmaps/sql/docmaps_index.sql

```diff
@@ -128,13 +128,25 @@ t_result AS (
     AND COALESCE(preprint_doi_and_url.preprint_doi, europepmc_response.doi) IS NOT NULL
 ),
 
-t_with_has_evaluations AS (
+t_tdm_doi_and_path AS(
   SELECT
-    *,
-    (ARRAY_LENGTH(evaluations) > 0) AS has_evaluations
+    DISTINCT
```

Comment:
Instead of using DISTINCT, could we select the latest record instead?
And maybe for now we would then also select the latest version? (maybe we could also use that as an _interim_ source of the version)

Reply:
Ok, can the tdm_path change in time?

Reply:
and will add the version info, good idea 👍